bazel-mode.el

```diff
@@ -60,6 +68,11 @@
       (delete-file buildifier-input-file)
       (delete-file buildifier-error-file))))
 
+(defun bazel-mode--buildifier-before-save-hook ()
+  "Run buildifer as a 'before-save-hook'."
```

Comment:
Emacs convention is `` `before-save-hook' `` (or with curly quotes if you prefer that).

Also nit: `before-save-hook` is the entire hook, so this should rather say "Function to be added to before-save-hook" or so.

Reply:
Done.